### PR TITLE
Create, Delete, Update question button

### PR DESCRIPTION
**Pull request by SimWPEric**
1. Create question 
image

2. Delete question trashcan with confirm modal 
image

3. Update question 
image

4. Duplicate question topic check for create and update 
image

### CODE REVIEW

**Review by XXJJXJ — APPROVED** [review]
LGTM

File: frontend/app/(dashboard)/questions/create/page.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      </FormControl>
+
+      <FormControl id="link" mb={4} isRequired>
+        <FormLabel>Link</FormLabel>
+        <Input name="link" value={link} onChange={handleLinkChange} />
+      </FormControl>
+
+      <Button colorScheme="teal" onClick={handleSubmit}>
+        Create
+      </Button>
```
Comment (keaganpzh) [inline frontend/app/(dashboard)/questions/create/page.tsx:125]: Actually would it be good to have a 'cancel' button that navigates the the previous page?

File: frontend/app/(dashboard)/questions/update/[id]/page.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+      </FormControl>
+
+      <FormControl id="link" mb={4} isRequired>
+        <FormLabel>Link</FormLabel>
+        <Input name="link" value={link} onChange={handleLinkChange} />
+      </FormControl>
+
+      <Button colorScheme="teal" onClick={handleSubmit}>
+        Update
+      </Button>
```
Comment (keaganpzh) [inline frontend/app/(dashboard)/questions/update/[id]/page.tsx:141]: Likewise for the update page

**Review by keaganpzh — COMMENTED** [review]
LGTM! Just a suggestion to add a cancel button

File: frontend/app/(dashboard)/questions/create/page.tsx (hunk 1 above)
Comment (SimWPEric) [reply frontend/app/(dashboard)/questions/create/page.tsx]: Good suggestion, added cancel button

image

File: frontend/app/(dashboard)/questions/update/[id]/page.tsx (hunk 2 above)
Comment (SimWPEric) [reply frontend/app/(dashboard)/questions/update/[id]/page.tsx]: Same as other comment

**Pull request merged**